src/main/java/com/synopsys/integration/alert/web/config/ConfigController.java

```diff
@@ -88,6 +94,9 @@ public ConfigController(final ConfigActions configActions, final ContentConverte
     @GetMapping("/{id}")
     public ResponseEntity<String> getConfig(@PathVariable final Long id) {
         final Optional<FieldModel> optionalModel;
+        if (!authorizationManager.hasAnyConfigReadPermissions()) {
```

Comment:
Dont we also need to know the context and descriptor name in order to properly handle permissions here? So if they dont have permission to the global email configuration but they do have permission to the global Black Duck configuration, then they would have access to retrieve both as long as they know the id's.
This will be relevant when User's can define their own roles/permission matrix